### ISSUE TITLE
Why is pry-nav discouraged for MRI?

### ISSUE THREAD

**Issue by justin808** [nixme/pry-nav#41]
Per the README:

> Using MRI? We recommend break or pry-byebug instead!

pry-byebug has some issue with zeitwerk

**Comment by iloveitaly** [comment]
@justin808 honestly, not sure. Want to submit a PR to change this?

Most likely, https://github.com/ruby/debug is going to be the path forward, but can't hurt to update this for now.

**Comment by nixme** [comment]
Agreed with @iloveitaly that ruby/debug will be the path forward, but I originally put the recommendation because `pry-nav` uses `set_trace_func` which at the time was much slower than byebug. Ruby docs also say `set_trace_func` is obsolete. We could change to the TracePoint API and remove that recommendation.

**Comment by justin808** [comment]
@iloveitaly Here you go: #42.
